pkg/apis/planetscale/v2/vitessshard_types.go

```diff
@@ -152,6 +152,12 @@ type VitessReplicationSpec struct {
 	//
 	// Default: true.
 	InitializeBackup *bool `json:"initializeBackup,omitempty"`
+
+	// ReplicationRepair specifies whether the replication repair should be executed
+	// when replica tablets have broken replication.
+	//
+	// Default: true.
+	ReplicationRepair *bool `json:"replicationRepair,omitempty"`
```

Comment:
So far, we've tried to keep the field names in VitessReplicationSpec as verb phrases, just for consistency. I'd also like to be more specific about what behavior this controls. Here's my suggestion, based on what the operator actually does when this is enabled:

```suggestion
	// RecoverRestartedMaster specifies whether the operator attempts to repair
	// replication when the master MySQL restarts in-place (due to a crash) or
	// its Pod gets deleted and recreated, causing the Pod IP to change.
	//
	// Default: true.
	RecoverRestartedMaster *bool `json:"recoverRestartedMaster,omitempty"`
```

Reply:
+1